creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -143,11 +170,25 @@ const updateReplicaSetJobProcessor = async function ({
     errorMsg = e.toString()
   }
 
+  // Make metrics to record
+  metricsToRecord = [
+    makeHistogramToRecord(
+      METRIC_NAMES[
+        `STATE_MACHINE_${QUEUE_NAMES.UPDATE_REPLICA_SET}_JOB_DURATION_SECONDS_HISTOGRAM`
+      ],
+      (Date.now() - startTimeMs) / 1000, // Metric is in seconds
+      {
+        result
+      }
+    )
+  ]
+
   return {
     errorMsg,
     issuedReconfig,
     newReplicaSet,
     healthyNodes,
+    metricsToRecord,
```

Comment:
could you dig into how this plays with the current logic for recording this metric?
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/stateMachineManager/processJob.ts#L71

Reply:
Why do we do metrics differently for this queue? I was basing these metrics off of the ones for IssueSyncRequest